src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --color-black: #000;
+  --color-blue: #00ACDC;
+}
+
+ul, li {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  margin: 0 auto;
+  padding-inline: 50px;
+}
+
+.header__wrap {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__logo {
+  display: block;
+}
+
+.nav ul {
+  display: flex;
+  list-style-type: none;
+}
+
+.nav li:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead